### PR TITLE
feat(refactor): remove findnodehandle, use function components & codegenNativeCommands

### PR DESCRIPTION
**Pull request by Titozzz**

### CODE REVIEW

File: android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java [hunk 1]
```diff
@@ -43,6 +43,7 @@
 import android.webkit.WebViewClient;
```
Comment (jamonholmgren) [inline android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java:43]: Is this using the legacy view manager compatibility layer?

**Review by jamonholmgren — COMMENTED** [review]
I've looked this over pretty thoroughly. It looks good, although there's still a lot I don't understand about how the new architecture works. For one thing, I'm curious why there are no changes on the iOS native side.

**Comment by Titozzz** [comment]
That's just some cleaning before real work gets done @jamonholmgren

**Comment by chiaramooney** [comment]
@Titozzz In order for Windows to still build following the upgrade to 0.68 please add 
`<WinUI2xVersion>2.8.0-prerelease.210927001</WinUI2xVersion>` to `windows\ExperimentalFeatures.props` and `example\windows\ExperinmentalFeatures.props`

**Comment by Titozzz** [comment]
@chiaramooney should I revert [090f8ad](https://github.com/react-native-webview/react-native-webview/pull/2508/commits/090f8adbc4d7962589b7dc85a4a2d996f8d95f8d) ?

**Comment by chiaramooney** [comment]
@Titozzz Yes! Tommy just created and merged the revert of his PR. So now you'll just need to refresh your branch and make the ExperimentalFeatures changes.

**Comment by Titozzz** [comment]
@chiaramooney It's already good in `windows\ExperimentalFeatures.props` and the other file does not exist, should I create it ?

**Comment by chiaramooney** [comment]
@Titozzz Yes please! The contents can match this file: https://github.com/microsoft/react-native-gallery/blob/main/windows/ExperimentalFeatures.props

File: src/WebViewShared.tsx [hunk 2]
```diff
@@ -86,3 +95,131 @@ export {
   defaultRenderLoading,
   defaultRenderError,
 };
+
+export const useWebWiewLogic = ({
```
Comment (TheAlmightyBob) [inline src/WebViewShared.tsx:99]: nit: typo

**Pull request merged**